Sources/AsyncHTTPClient/HTTPClientProxyHandler.swift

```diff
@@ -14,6 +14,7 @@
 
 import NIO
 import NIOHTTP1
+import NIOSOCKS
 
 public extension HTTPClient.Configuration {
```

Comment:
Can we delete the word `public` from here and move it to `struct Proxy`? It's making it hard to reason about the visibility of things.

Reply:
Not really, I didn't add this, so it will break the public API.

Reply:
No it won't. The only thing the `public` applies to here is the `struct`.

Reply:
Ah, understood.